app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -333,11 +332,17 @@ class NavigationDrawerTestActivityTest {
 
   @Test
   // TODO(#973): Fix NavigationDrawerTestActivityTest
-  @Ignore
   fun testNavigationDrawerTestActivity_withAdminProfile_openNavigationDrawer_checkAdministratorControlsDisplayed() { // ktlint-disable max-line-length
-    launch<NavigationDrawerTestActivity>(createNavigationDrawerActivityIntent(0)).use {
-      onView(withContentDescription(R.string.drawer_open_content_description)).perform(click())
-      onView(withId(R.id.administrator_controls_linear_layout)).check(matches(isDisplayed()))
+    launch(NavigationDrawerTestActivity::class.java).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withContentDescription(R.string.drawer_open_content_description)).check(
+        matches(isCompletelyDisplayed())
+      ).perform(click())
+      onView(
+        allOf(
+          withText(R.string.administrator_controls)
```

Comment:
Can we confirm here that if we launch the intent with the non-admin profile, this test must fail?